cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Committee.hs

```diff
@@ -59,6 +61,7 @@ data GovernanceCommitteeCreateColdKeyResignationCertificateCmdArgs era =
   GovernanceCommitteeCreateColdKeyResignationCertificateCmdArgs
     { eon               :: !(ConwayEraOnwards era)
     , vkeyColdKeySource :: !(VerificationKeyOrHashOrFile CommitteeColdKey)
+    , anchor            :: !(Maybe (Ledger.Anchor (Ledger.EraCrypto (ShelleyLedgerEra era))))
```

Comment:
Have a look at how the governance actions parsers handle anchors. E.g
```
data GovernanceActionCreateNoConfidenceCmdArgs era
  = GovernanceActionCreateNoConfidenceCmdArgs
      { eon                   :: !(ConwayEraOnwards era)
      , networkId             :: !Ledger.Network
      , deposit               :: !Lovelace
      , returnStakeAddress    :: !AnyStakeIdentifier
      , proposalUrl           :: !ProposalUrl
      , proposalHashSource    :: !ProposalHashSource
      , governanceActionId    :: !TxId
      , governanceActionIndex :: !Word32
      , outFile               :: !(File () Out)
      } deriving Show
```
In particular the `proposalUrl` and `proposalHashSource` fields.

Reply:
So you prefer to flatten these two fields , url and hash. 
But what to do about `ProposalUrl` and `ProposalHashSource` which  are not accurately describing this Anchor, no - since it's not related to proposal, but to resignation..  Create new types for them?

Reply:
So you can construct an `Anchor` as follows: https://github.com/input-output-hk/cardano-cli/blob/e86aa60f4481dcdc035d2173d61090bd284ad089/cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Actions.hs#L98

Reply:
Hm, I don't understand exactly what the suggestion is (probably because I'm not familiar with the codebase). 
Would you like me to represent Anchor  in  `GovernanceCommitteeCreateColdKeyResignationCertificateCmdArgs` as  two different fields, similar to the example you gave above, so as : 

```
   ,    resignationlUrl           :: !ProposalUrl
      , resignationHashSource    :: !ProposalHashSource
```

instead of the field that I have added: 
```
, anchor            :: !(Maybe (Ledger.Anchor (Ledger.EraCrypto (ShelleyLedgerEra era))))
```

? 

Did i understand correctly?

Reply:
Yes exactly :+1:

Reply:
Both of them will have to be `Maybe` I suppose, since the field is optional.  I hope it won't be to messy, if say one is there and the other one not  :thinking: